File: problem_4/problem_4.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int size = nums.length;
        if(size<=2)
        {
            return size;
        }
        int k = 2;
        for(int i=0;i<size;i++)
        {
            if(nums[i] != nums[k-2])
            {
                nums[k] = nums[i];
                k++;
            }
        }
        return k;
    }
}

File: problem_4/problem_4.py
class Solution(object):
    def removeDuplicates(self, nums):
        size = len(nums)
        if size <= 2:
            return size
        k = 2
        for i in range(2, size):
            if nums[i] != nums[k-2]:
                nums[k] = nums[i]
                k = k + 1
        return k
